services/showcase/api/services/sandbox.py

```diff
@@ -230,3 +236,20 @@ async def accept_invitation(
         sender_id=sender.id,
         connection_id=uuid.UUID(resp["connection_id"]),
     )
+
+
+async def promote_tenant_to_issuer(
+    sandbox_id: uuid.UUID,
+    tenant_id: uuid.UUID,
+    db: AsyncSession,
+) -> AcceptInvitationResponse:
+    sandbox = await get_sandbox(sandbox_id, db)
+    tenant = await get_tenant(sandbox, tenant_id, db)
+
+    # have tenant register itself as issuer
+    resp = await traction.tenant_admin_issuer(tenant.wallet_id, tenant.wallet_key, {})
+    logging.error(resp.status_code)
```

Comment:
logging/logger is not imported and declared

Reply:
i don't know why, but my `tox lint` is not catching these.